llvm/test/CodeGen/AMDGPU/true16-copy-vgpr16-to-sgpr32.mir

```diff
@@ -0,0 +1,118 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 3
+# RUN: llc -march=amdgcn -mcpu=gfx1100 -mattr=+real-true16 -run-pass=si-fix-sgpr-copies -verify-machineinstrs -o - %s | FileCheck %s
+
+# Ensure READFIRSTLANE is generated, and that its src is REG_SEQUENCE.
+
+---
```

Comment:
Test is larger than it needs to be

Reply:
Hi Matt. Do you suggest to simplify the test code and make it smaller?

Reply:
Compact register numbers with -run-pass=none. Also avoid the copy of undef, that's going to stress other weird edge cases and is equivalent to IMPLICIT_DEF. Can also do some pre-folding of operands.


But for this an IR test that hits the case would be long term more reliable, both would be useful